src/keyszer/config_api.py

```diff
@@ -41,6 +41,27 @@
 # multipurpose timeout
 _TIMEOUTS = TIMEOUT_DEFAULTS
 
+# global dict of delay values used to mitigate Unicode entry sequence and macro or combo failures
+THROTTLE_DELAY_DEFAULTS = {
+    'key_pre_delay_ms': 0,
+    'key_post_delay_ms': 0,
+}
+_THROTTLES = THROTTLE_DELAY_DEFAULTS
+
+def throttle_delays(key_pre_delay_ms=0, key_post_delay_ms=0):
+    _kpre, _kpost, ms_min, ms_max = key_pre_delay_ms, key_post_delay_ms, 0.0, 150.0
+    if ms_min <= _kpre <= ms_max: _THROTTLES.update({'key_pre_delay_ms': _kpre})
+    else: error(f'throttle_delays(): key_pre_delay_ms must be {ms_min} to {ms_max} ms. Defaulting to 0 ms.')
+    if ms_min <= _kpost <= ms_max: _THROTTLES.update({'key_post_delay_ms': _kpost})
+    else: error(f'throttle_delays(): key_post_delay_ms must be {ms_min} to {ms_max} ms. Defaulting to 0 ms.')
+    # Show values in log if user sets custom delays
+    if any(_THROTTLES.values()):
+        debug(f'THROTTLES: {key_pre_delay_ms = }, {key_post_delay_ms = }')
+
+# for use with throttle delays
+def sleep_ms(msec):
```

Comment:
This can be moved directly into output now (or do we have a utils?), as we'll never way to immediately sleep in the API config...

Reply:
Working on it.